pkgs/development/compilers/haxe/haxe-packages.nix

```diff
@@ -0,0 +1,274 @@
+{ stdenv, fetchzip }:
+
+let
+  withCommas = stdenv.lib.replaceChars ["."] [","];
+in {
+  hxcpp = stdenv.mkDerivation rec {
+    libname = "hxcpp";
+    version = "3.4.64";
+    name = "${libname}-${version}";
+
+    src = fetchzip {
+      inherit name;
+      url = "http://lib.haxe.org/files/3.0/${withCommas name}.zip";
```

Comment:
`3.0` is some kind of framework number, isn't it? Would it be worth to make it a variable?